modules/dkan/dkan_periodic_updates/dkan_periodic_updates.module

```diff
@@ -408,6 +416,91 @@ function dkan_periodic_updates_state() {
   return $output;
 }
 
+
+/**
+ * Page callback for periodic updates reset page.
+ *
+ * @param $form
+ * @param $form_state
+ *
+ * @return mixed
+ */
+function dkan_periodic_updates_reset_form($form, &$form_state) {
+  $active = variable_get('dkan_periodic_updates_status');
+  if ($active) {
+    $file = variable_get('dkan_periodic_updates_manifest');
+    $file = file_load($file);
+    if ($file) {
+      $handle = fopen($file->uri, 'r');
+      $manifest_options = [];
+      $headers = fgetcsv($handle, 0, ',');
+      while (($data = fgetcsv($handle, 0, ",")) !== FALSE) {
+        $resource_data = array_combine($headers, $data);
+        if (!empty(variable_get('dkan_periodic_updates_' . $resource_data['resource_id'], ''))) {
```

Comment:
On a fresh install when no resource has been updated this looks a little bit broken because it shows the dropdown but without options, I think we can choose if we want to:
1- hide the form and say there is nothing to reset because no resource has been updated yet, or
2- change the condition in order to load all the options.

I'll leave this up to you.